.env.example

```
@@ -1,6 +1,6 @@
 DJANGO_SECRET_KEY=DJANGO_SECRET_KEY
 DJANGO_DEBUG=True
-ALLOWED_HOSTS=16.170.148.253,localhost,127.0.0.1
+ALLOWED_HOSTS=13.61.159.130,127.0.0.1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ALLOWED_HOSTS key should go before the DJANGO_DEBUG key

(UnorderedKey)